changelog/@unreleased/pr-6835.v2.yml

```
@@ -0,0 +1,6 @@
+type: improvement
+improvement:
+  description: Whenever a new version is published, we also tag our container images
+    to antithesis reposity. And schedule an experiment to run regularly now.
```

```suggestion
  description: We now tag our container images to the Antithesis repository whenever a new version of AtlasDB is published. Additionally, CI now schedules an Antithesis experiment at 10am each day.
```

If you choose to have a changelog here, then let's be more explicit (and fixed a typo in repository, unless it's supposed to be reposity!)